tsconfig.json

```diff
@@ -15,9 +16,10 @@
     "resolveJsonModule": true,
     "outDir": "lib",
     "rootDir": "src",
+    "skipLibCheck": true,
```

Comment:
Wondering if we should track removing this field at some point, as it usually better practice to not have to use it.

Reply:
What change in particular required enabling `skipLibCheck`?

Reply:
Without it I see complaints about `@rjsf`:
```
node_modules/@rjsf/utils/lib/types.d.ts:102:37 - error TS2339: Property 'autocomplete' does not exist on type 'HTMLInputElement'.

102     autoComplete?: HTMLInputElement['autocomplete'];
                                        ~~~~~~~~~~~~~~

node_modules/@rjsf/utils/lib/types.d.ts:735:37 - error TS2339: Property 'autocomplete' does not exist on type 'HTMLInputElement'.

735     autocomplete?: HTMLInputElement['autocomplete'];
                                        ~~~~~~~~~~~~~~

node_modules/yjs/dist/src/types/YXmlFragment.d.ts:163:36 - error TS2304: Cannot find name 'Node'.

163     } | undefined, binding?: any): Node;
                                       ~~~~
```
I assume it appeared because I added use of JupyterLite `DriveFS` and that brings in more dependencies, but I don't know.

I would rather not use the option, but I couldn't find a quick fix.

Reply:
Opened https://github.com/jupyterlite/cockle/issues/41 to track this.